etc/ci/upload_nightly.sh

```diff
@@ -41,7 +46,7 @@ main() {
     # Lack of quotes on package allows glob expansion
     # Note that this is not robust in the case of embedded spaces
     # TODO(aneeshusa): make this glob robust using e.g. arrays or Python
-    upload "${platform}" ${package}
+    upload "${platform}" ${package} ${extension}
```

Comment:
The `${extension}` parameter should be quoted because it is not performing a glob. See our [style guide](https://github.com/servo/servo/blob/master/docs/STYLE_GUIDE.md).